app/assets/stylesheets/custom.css.scss

```diff
@@ -227,13 +227,18 @@ p.copyright {
     opacity: 0;
     filter: alpha(opacity=0);
 }
+
 .q-width{
   margin: auto;
   width: 40%;
   min-width: 300px;
 }
-.row .btn{
-  margin-bottom: 1em;
+
+.btn-margin{

```

Comment:
It’s I’m asking actually ))
so for you it looks logical and you’re ok with this?

From: Orest Falchuk [mailto:notifications@github.com]
Sent: Wednesday, December 16, 2015 6:37 PM
To: if060-webui-RoR/Aid-Interview <Aid-Interview@noreply.github.com>
Cc: Sergiy Moskovchuk <smoskov@softserveinc.com>
Subject: {Disarmed} Re: [Aid-Interview] As an Admin I should not be able to remove topic if it has any questions (#55)


In app/assets/stylesheets/custom.css.scss<https://github.com/if060-webui-RoR/Aid-Interview/pull/55#discussion_r47799017>:

>  .q-width{

>    margin: auto;

>    width: 40%;

>    min-width: 300px;

>  }

> -.row .btn{

> -  margin-bottom: 1em;

> +

> +.btn-margin{

It will be OK if i just change css class name for btn-margin ?

—
Reply to this email directly or view it on GitHub<https://github.com/if060-webui-RoR/Aid-Interview/pull/55/files#r47799017>.